source/css/sass/_grid.scss

```diff
@@ -14,4 +15,6 @@
     --GRID-COLUMN-GAP: #{($grid-max_width * $grid-column_gap) / 100%};
     --GRID-COLUMN-WIDTH: calc((#{$grid-max_width} - (var(--GRID-COLUMN-GAP) * #{$grid-main_column_count - 1})) / #{$grid-main_column_count});
   }
+
+  @include logical-property(min, inline-size, $grid-min_width, $to-rem: false);
```

Comment:
Didn't use `min-inline-size()` to avoid rem, though probably not a problem? 🤔

Reply:
lgtm